Gemfile

```
@@ -23,7 +23,7 @@ group :test do
   gem "nokogiri", "~> 1.7"
   gem "rspec"
   gem "rspec-mocks"
-  gem "rubocop", "~> 1.8.1"
+  gem "rubocop", "~> 1.9"
```

I think it would be better if we continue having a version constraint to avoid false positives when rubocop-1.10 releases in future.
```suggestion
  gem "rubocop", "~> 1.9.1"
```